changelog.d/20241107_185222_danyal.faheem_remove_py3_8_reference.md

```diff
@@ -0,0 +1 @@
+- [Depreciation] Drop support for python 3.8 as it has reached end of life. (by @Danyal-Faheem)
```

Comment:
nit: In changelog template, its `Deprecation`, not `Depreciation`

Reply:
Whoops, thanks for picking it out.